docs/query-dsl/bool-dsl/operators/and-assign-many-manual-bools-usage.asciidoc

```diff
@@ -0,0 +1,160 @@
+:ref_current: https://www.elastic.co/guide/en/elasticsearch/reference/2.3
+
+:github: https://github.com/elastic/elasticsearch-net
+
+:nuget: https://www.nuget.org/packages
+
+////
+IMPORTANT NOTE
+==============
+This file has been generated from https://github.com/elastic/elasticsearch-net/tree/2.x/src/Tests/QueryDsl/BoolDsl/Operators/AndAssignManyManualBoolsUsageTests.cs. 
```

Comment:
LGTM - the docs need to be regen'ed to get the correct version and reference link